pkg/infra/process/root_check_windows.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
+
+package process
+
+func isServerRunningAsRoot() (bool, error) {
+	// TODO implement Windows process root check
```

Comment:
https://github.com/golang/go/issues/28804#issuecomment-505326268 - this might bring some clues for Windows solution.

Reply:
Yep that was pretty much the commit I reverted. For another day 🤞